composer.json

```diff
@@ -1,5 +1,39 @@
 {
+    "name": "allegro/php-protobuf",
+    "description": "Google's Protocol Buffers for PHP",
+    "type": "library",
+    "keywords": [
+        "protocol buffers",
+        "data",
+        "pb"
+    ],
+    "homepage": "https://github.com/allegro/php-protobuf",
+    "bin": ["bin/protoc"],
```

Comment:
The `bin/protoc` doesn't exist. If anything the `protoc-gen-php.php` should be added as a vendor binary. Simple listing it here doesn't work.